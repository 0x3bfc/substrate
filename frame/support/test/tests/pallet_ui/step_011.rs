#[frame_support::pallet]
mod pallet {
	#[pallet::config]
	pub trait Config: frame_system::Config {}

	#[pallet::pallet]
	pub struct Pallet {}
}

fn main() {
}
