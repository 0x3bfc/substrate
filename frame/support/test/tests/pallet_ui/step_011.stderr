error: Invalid type def generics: expect `T` or `T, I = ()`
 --> $DIR/step_011.rs:7:13
  |
7 |     pub struct Pallet {}
  |                ^^^^^^

error: unexpected end of input, expected `T`
 --> $DIR/step_011.rs:1:1
  |
1 | #[frame_support::pallet]
  | ^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = note: this error originates in an attribute macro (in Nightly builds, run with -Z macro-backtrace for more info)
